brew/kubestellar-0.6.0.rb

```diff
@@ -0,0 +1,107 @@
+class Kubestellar060 < Formula
+  desc "KubeStellar is a flexible solution for challenges associated with multi-cluster configuration management for edge, multi-cloud, and hybrid cloud"
+  homepage "https://kubestellar.io"
+
+  if OS.mac?
+    case Hardware::CPU.arch
+    when :x86_64
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_darwin_amd64.tar.gz"
+      sha256 "310f6724555da8e243d624c6519989acb1b95583a5d0657883f497b64d63d375"
+    when :arm
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_darwin_arm64.tar.gz"
+      sha256 "590f5867a6bba6d8dbfc93b5c4c19c3acbbc592df8544ef48dcd737cceab443b"  
+    else
+      odie "Unsupported architecture on macOS"
+    end
+  elsif OS.linux?
+    case Hardware::CPU.arch
+    when :x86_64
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_linux_amd64.tar.gz"
+      sha256 "14aaeb105c77b2e8466c48553d2f1b97394fd5b27caaf7358adcc9e7108de247"
+    when :arm
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_linux_arm64.tar.gz"
+      sha256 "bb80d33d1a62fe5b3aaf0cecee2f12ec92e2e78d5a0f5678eef08996ba29ce71"
+    when :s390x
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_linux_s390x.tar.gz"
+      sha256 "303937fa034bb307e6db8f6202ac3f6a79dd9ffa22d8f7269569bd8d723a804e"
+    when :ppc64
+      url "https://github.com/kubestellar/kubestellar/releases/download/v0.6.0/kubestellar_v0.6.0_linux_ppc64le.tar.gz"
+      sha256 "0b9ec309912a0d74a562e8dbe6b9e0fa9ed0384cb08ad132cebb9293a7b11ca0"
+    else
+      odie "Unsupported architecture on Linux"
+    end
+  else
+    odie "Unsupported operating system"
+  end
+
+  license "Apache-2.0"
+
+  depends_on "kubectl"
+  depends_on "yq"
+  depends_on "jq"
+
+  def install
+    if OS.mac?
+      case Hardware::CPU.arch
+      when :x86_64
+        home_directory = File.join(ENV["HOMEBREW_PREFIX"])
+        target_directory = File.join(home_directory, 'kubestellar')
+        # puts "HELLO THERE #{target_directory}"
+
+        bin.install Dir["bin/*"]
+
+        system "osascript", "-e", <<-EOS
+          do shell script "mkdir -p '#{target_directory}'" with administrator privileges
+          do shell script "cp -r config user LICENSE README.md '#{target_directory}'" with administrator privileges
+        EOS
+      when :arm
+        home_directory = File.join(ENV["HOMEBREW_PREFIX"])
+        target_directory = File.join(home_directory, 'kubestellar')
+        # puts "HELLO THERE #{target_directory}"
+
+        bin.install Dir["bin/*"]
+
+        system "osascript", "-e", <<-EOS
+          do shell script "mkdir -p '#{target_directory}'" with administrator privileges
+          do shell script "cp -r config user LICENSE README.md '#{target_directory}'" with administrator privileges
```

Comment:
This looks identical to the other case?  Do we need a case statement here?

Reply:
removed